cmd/sqlbee/mutate.go

```diff
@@ -123,6 +124,14 @@ func mutatePodSpec(volumes []corev1.Volume, proxyContainer *corev1.Container, po
 			podSpec.Containers = append(podSpec.Containers[:i], podSpec.Containers[i+1:]...)
 			break
 		}
+		container.Resources.Limits = corev1.ResourceList{
```

Comment:
Do we really want to set resource limits in the pod specification? Some time ago we removed all hard limits, due to problems with the k8s cqueue configuration causing huge delays, especially if milli-CPUs are used.
Do we have resource limits in other services?

Reply:
I share the same concerns. Unless we have verified that limits no longer lead to undesired execution pauses we should remove them and only specify requested resources

Reply:
We have resource limits on istio proxy sidecar for example: 
```yaml
    resources:
      limits:
        cpu: "2"
        memory: 1Gi
      requests:
        cpu: 100m
        memory: 128Mi
```
The limits are not in mCPU so they may not cause CPU throttling. This is the issue regarding that https://github.com/kubernetes/kubernetes/issues/67577 and still not clear yet if it's resolved or not. 

But one reason I'd like to use limits is for provisioning, it'll be a simple math of summing the limits of the pods. In the other hand if we don't specify it then we have less control on the resources. 

I can of course  remove the changes limiting the resources until we are sure about it.

Reply:
I would prefer to remove the hard limits for now. They enable CFS quotas, which we currently don't want to use.
We can make this configurable in the future, because I also think that the limits are useful. But right now k8s is too buggy in this regard.